pages/common/xxd.md

```diff
@@ -0,0 +1,23 @@
+# xxd
+
+> Create a hexdump from a binary file or vice-versa.
+
+- Generate a hexdump from a binary file and display the output to console:
+
+`xxd {{input_file}}`
```

Comment:
I believe you can pipe to this command too. Not that it needs adding here - just a passing observation.